.travis.yml

```diff
@@ -38,11 +38,34 @@ install:
   - python setup.py install build_ext -i
   - pip install codecov
 
+before_script:
+- if [ "${TEST}" == "extra" ]; then
+        mkdir ~/repo-clone && cd ~/repo-clone &&
```

Comment:
add these two lines at the beginning of this if block: https://github.com/pyqtgraph/pyqtgraph/blob/develop/.travis.yml#L36-L37